modules/meilisearch/meilisearch_test.go

```diff
@@ -0,0 +1,86 @@
+package meilisearch_test
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/meilisearch"
+)
+
+func TestMeilisearch(t *testing.T) {
+	ctx := context.Background()
+
+	ctr, err := meilisearch.Run(ctx, "getmeili/meilisearch:v1.10.3")
+	testcontainers.CleanupContainer(t, ctr)
+	require.NoError(t, err)
+
+	address, err := ctr.Address(ctx)
+	require.NoError(t, err)
+
+	client := http.DefaultClient
+
+	req, err := http.NewRequest(http.MethodGet, address, nil)
+	require.NoError(t, err)
+
+	resp, err := client.Do(req)
```

Comment:
nit: no need for a specific variable here
```suggestion
	req, err := http.NewRequest(http.MethodGet, address, nil)
	require.NoError(t, err)

	resp, err := http.DefaultClient.Do(req)
```